core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -2016,6 +2016,7 @@ components:
     EntityModule:
       type: string
       enum:
+        - BootLoader
```

Comment:
Very minor, but we should add this to the bottom so it aligns with the partition ordering. We should also update https://radixdlt.atlassian.net/wiki/spaces/S/pages/2979430761/REP-60+Persisted+State+Model - which number did we choose for boot loader partition? 0x10 to leave room for some more modules?

Reply:
Good point, 0x20 (32) was chosen to leave room for more modules